data/saas/dataset/saas_example_dataset.yml

```diff
@@ -224,3 +224,11 @@ dataset:
           - name: mailing_list_name
             fidesops_meta:
               data_type: string
+  - fides_key: saas_connector_external_example
+    name: An Example External SaaS Dataset
+    description: An example "external" dataset used for a SaaS connector
+    collections:
+      - name: customer_id_reference_table
+        fields:
+          - name: customer_id
+            data_categories: [system.operations]
```

Comment:
yeah, it seems like it's _not_ allowed with saas configs. 

i can split this out into another dataset yml file if you think it's clearer, i was on the fence about whether it's clearer to do that or to limit testing file/artifact sprawl and just put it in the same file as the original dataset, since it's really only useful in relation to that original dataset

Reply:
I think it's okay, I just hadn't seen it before!